src/concrete/ml/sklearn/base.py

```diff
@@ -2069,22 +2105,53 @@ def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
         Returns:
             numpy.ndarray: The majority vote.
         """
-        y_preds_processed = []
-        for y in y_preds:
-            vote = self.majority_vote(y.flatten())
-            y_preds_processed.append(vote)
+        return numpy.array([self.majority_vote(y.flatten()) for y in y_preds])
 
-        return numpy.array(y_preds_processed)
+    def get_topk_labels(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
+        """Return the K-nearest labels of a point.
 
-    def predict(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
+        Args:
+            X (Data): The input values to predict, as a Numpy array, Torch tensor, Pandas DataFrame
+                or List.
+            fhe (Union[FheMode, str]): The mode to use for prediction.
+                Can be FheMode.DISABLE for Concrete ML Python inference,
+                FheMode.SIMULATE for FHE simulation and FheMode.EXECUTE for actual FHE execution.
+                Can also be the string representation of any of these values.
+                Default to FheMode.DISABLE.
+
+        Returns:
+            numpy.ndarray: The topk labels for each point.
+        """
 
         X = check_array_and_assert(X)
 
         topk_labels = []
         for query in X:
             topk_labels.append(BaseEstimator.predict(self, query[None], fhe=fhe))
 
-        y_preds = self.post_processing(numpy.array(topk_labels))
+        return numpy.array(topk_labels).squeeze()
+
+    def predict(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
+        """Predict values for X, in FHE or in the clear.
```

Comment:
you don't need to put docstrings for `predict` (since the class inherits from BaseEstimator !